scripts/rpc.py

```diff
@@ -341,6 +341,49 @@ def bdev_ocf_set_cache_mode(args):
     p.add_argument('mode', help='OCF cache mode', choices=['wb', 'wt', 'pt', 'wa', 'wi', 'wo'])
     p.set_defaults(func=bdev_ocf_set_cache_mode)
 
+    def bdev_ocf_set_cleaning_alru(args):
+        print_json(rpc.bdev.bdev_ocf_set_cleaning_alru(args.client,
+                                                       name=args.name,
+                                                       wake_up=args.wake_up,
+                                                       staleness_time=args.staleness_time,
+                                                       flush_max_buffers=args.flush_max_buffers,
+                                                       activity_threshold=args.activity_threshold))
+    p = subparsers.add_parser('bdev_ocf_set_cleaning_alru',
+                              help='Set ALRU cleaning policy with parameters on OCF cache device')
+    p.add_argument('name', help='Name of OCF bdev')
+    p.add_argument('-w', '--wake-up', type=int, default=-1,
```

Comment:
I think it might be better to set the default implicitly to 20 here (and other default values too) and not base it on defaults in OCF? In that case changing the type to uint32 would also be useful, I think.

Reply:
I think there are two problems with your proposed approach:
1. SPDK maintainers prefer to be as independent as possible from any hard-coded values. This way, when we change OCF defaults in the future, there will be no need to update them also in SPDK adapter.
2. Some of those parameters takes a range from 0 to some value, so the default=-1 informs of no change to that particular parameter. I think it will be much more obfuscated to handle this situation using different type of variable here.